core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -201,8 +201,12 @@ func (eb *EthBroadcaster) monitorEthTxs(k ethkey.Key, triggerCh chan struct{}) {
 	}
 }
 
-func (eb *EthBroadcaster) ProcessUnstartedEthTxs(key ethkey.Key) error {
-	return eb.advisoryLocker.WithAdvisoryLock(context.TODO(), postgres.AdvisoryLockClassID_EthBroadcaster, key.ID, func() error {
+func (eb *EthBroadcaster) ProcessUnstartedEthTxs(key ethkey.KeyV2) error {
+	keyState, err := eb.keystore.GetState(key.ID())
```

Comment:
Can we call it `GetStateID` since `GetState` mistakenly implies that it might be up to date?

Reply:
that's a good call - how about `GetStateRecord()`? Since it returns a struct, rather than an ID.

Reply:
Update: I see your comment below now

Reply:
I actually think we can drop GetState entirely. BPTXM can update using the address as pkey.